src/types/index.ts

```diff
@@ -187,3 +187,38 @@ export interface MarketSnapshot {
   totalSupply: string;
   updatedAt: string;
 }
+
+export type TransactionEvents =
```

Comment:
The type name should not be pluralized
```suggestion
export type TransactionEvent =
```

Reply:
I think `TransactionEvents` is more accurate. because it is a union of all possible events

Reply:
Even if it's a union, at the end it only corresponds to one possible values out of all the options it accepts